docs/integration/social-networks.md

```diff
@@ -8,28 +8,34 @@ You can set up which social network logins are enabled on which domain in `socia
 
 For every social network, you need `app_id` and `app_secret`. Below you have information on where and how to get them. The credentials then need to be set to the corresponding environment variables.
 
+Every social network require URL for backward redirect to you app.
+In Shopsys platform, you have to add url `{eshop_domain}/social-network/login/{type}` where `{eshop_domain}` is your domain and `{type}` is social network name, e.g.: `google`, `facebook`, `seznam`, etc.
+For local development, social networks usually require https.
+
 ### Facebook
 
 -   go to site https://developers.facebook.com/ and log into Facebook account
 -   create [application](https://developers.facebook.com/apps)
 -   in that application, in `App settings` → `Basic` you will find `app_id` and `app_secret`
 
+P.S.: If you are owner of the Facebook application, and you log in with this owner account, email will not come after login, more info: https://github.com/hybridauth/hybridauth/issues/1372#issuecomment-1542447803
+
 ### Google
 
 -   log into Google account and create [OAuth 2.0 Client IDs](https://console.cloud.google.com/apis/credentials)
+    -   first, you need to create a project,
```

Comment:
i'm not sure whether items in the bullet list should end with the comma

Reply:
Interesting, I thought this is correct grammar but probably just in Czech. Rewritten in 50bb802b0e

Reply:
ok, that rendered terribly :sweat_smile: 
![image](https://github.com/user-attachments/assets/fcacaf83-8406-493e-93c7-c9f137c5664d)

Changed back to the unordered list in d762a365e3 :smile: